### PR TITLE
[Enhancement] Add uptime statistic and suggestion for Lashing Flames

### PR DESCRIPTION
**Pull request by darya-talanina**
Related to https://github.com/WoWAnalyzer/WoWAnalyzer/issues/3222

Screenshot 2020-11-05 at 23 56 08
Screenshot 2020-11-06 at 00 20 39

### CODE REVIEW

File: src/parser/shaman/enhancement/modules/talents/LashingFlames.tsx [hunk 1]
```diff
@@ -51,6 +70,24 @@ class LashingFlames extends Analyzer {
...
+          try to maintain 100% uptime for maximum damage increase.
+          To achieve this, you can strike the target with <SpellLink id={SPELLS.LAVA_LASH.id} />{` `}
+          when <SpellLink id={SPELLS.LASHING_FLAMES_DEBUFF.id} /> is about to expire.
+        </>
+      )
+    )
+      .icon(SPELLS.LASHING_FLAMES_TALENT.icon)
+      .actual(<><SpellLink id={SPELLS.LASHING_FLAMES_DEBUFF.id} /> was active for {(actual*100).toFixed(2)}% of the fight</>)
+      .recommended('recommended: 99%')
+    )
```
Comment (Pewtro) [inline src/parser/shaman/enhancement/modules/talents/LashingFlames.tsx:88]: You can use ``formatPercentage(actual,)`` for the actual as it defaults to 2 decimal precision, and you can use ``recommended`` for the recommended value like ``.recommended(>{formatPercentage(recommended)}% is recommended)``

Comment (darya-talanina) [reply src/parser/shaman/enhancement/modules/talents/LashingFlames.tsx:88]: @Pewtro you're right :+1:. I implemented your recommendation.

**Comment by Pewtro** [comment]
Looks good to me, and is mergable once the checks finish!

**Comment by Pewtro** [comment]
Thanks

**Pull request merged**